includes/HelpCenterFeatureHooks.php

```diff
@@ -28,6 +29,32 @@ public function __construct() {
 	public function hooks() {
 		// add filter so we don't show/load help during onboarding
 		add_filter( 'newfold/features/filter/isEnabled:helpCenter', array( $this, 'filterValue' ) );
+		add_filter( 'newfold/features/action/onDisable:helpCenter', array( $this, 'clearhelpCenter' ) );
+	}
+
+	/**
+	 * Enqueue inline script as back up to hide the helpcenter UI on onboarding flow
+	 */
+	public function clearhelpCenter() {
+		add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_clear_storage_script' ), 20 );
+	}
+
+	/**
+	 * Inline script to clear localStorage
+	 */
+	public function enqueue_clear_storage_script() {
+
+		$clear_storage_js = <<<JS
+			document.addEventListener("DOMContentLoaded", function() {
+				const nfdHelpContainer = document.getElementById( 'nfd-help-center' );
+				nfdHelpContainer.classList.toggle( 'help-container', false );
+			});
+		JS;
+
+		// Add the inline script to `nfd-help-center` if it’s enqueued
+		if ( wp_script_is( 'nfd-help-center', 'enqueued' ) ) {
```

Comment:
Just wanted to check, this always remains enqueued right?
Also, https://github.com/newfold-labs/wp-module-help-center/blob/a09d1cf6bf5c95bc93d79842f8c46ce3472caef2/includes/HelpCenter.php#L26